manager/controllers/app/m4dapplication_controller.go

```diff
@@ -102,25 +110,25 @@ func (r *M4DApplicationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 	return ctrl.Result{}, nil
 }
 
-func checkBlueprintStatus(applicationContext *app.M4DApplication, blueprint *app.Blueprint) {
+func checkBlueprintStatus(applicationContext *app.M4DApplication, status ResourceStatus) {
```

Comment:
Should this be called checkResourceStatus since it checks blueprint or plotter?

Reply:
Updated the function name.